src/dependency.cr

```diff
@@ -57,6 +53,21 @@ module Shards
       end
     end
 
+    def as_package
```

Comment:
```suggestion
    def as_package : Package?
```

for clarity, but as per another comment this method might not be needed.

Reply:
Yeah... maybe the method should be named `as_package?` to make it even more clear